src/python-common/ceph/deployment/drive_selection/selector.py

```diff
@@ -34,6 +37,17 @@ def __init__(self,
             self._db = self.assign_devices(self.spec.db_devices)
             self._journal = self.assign_devices(self.spec.journal_devices)
 
+    def find_osds_in_spec(self):
+        # type: () -> List["DaemonDescription"]
+        osds = []  # type: List["DaemonDescription"]
```

Comment:
```
_________________ ceph/deployment/drive_selection/selector.py __________________
20: error: Name 'DaemonDescription' is not defined
40: error: Name 'DaemonDescription' is not defined
42: error: Name 'DaemonDescription' is not defined
```

I fear, this is'n going to work this way. What about just cut&paste this method into mgr/orchestrator/_interface.py?